packages/doc-contribs/pr-antora-content-guidelines-checker/src/main.ts

```diff
@@ -36,9 +41,18 @@ async function run(): Promise<void> {
     const forbiddenPatternToCheckInput = core
       .getInput("forbidden-pattern-to-check")
       .split(",");
+
+    const stepToSkip: string[] = core
```

Comment:
**polish:** the are several steps, so the constant could be renamed into `stepsToSkip`